### PR TITLE
Fix Error in the Courses link Issue 415

### PR DESCRIPTION
**Pull request by jciasenza**
Fixes https://github.com/openedx/frontend-app-learner-dashboard/issues/415

What changed?
frontend-app-learner-dashboard/src/containers/LearnerDashboardHeader/LearnerDashboardMenu.jsx

in line 16 
remplace 
href ='/'
by 
href =''

image

If something needs to be modified, let me know, thank you!!!
Atte
Juan Carlos (Aulasneo)

Developer Checklist

Test suites passing
Documentation and test plan updated, if applicable
Received code-owner approving review

    Testing Instructions

[ How should a reviewer test this PR? ]

Reviewer Checklist

Collectively, these should be completed by reviewers of this PR:

I've done a visual code review

    I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jciasenza!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@2U-aperture`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by MaxFrank13** [comment]
Hey @jciasenza it looks like this is no longer passing CI and needs to be updated with the latest from the `master` branch

**Comment by jciasenza** [comment]
I review it

**Comment by jciasenza** [comment]
I made a new push with some test changes, because in production it seems that the Courses link works well, that is to say that I made that change back

**Comment by itsjeyd** [comment]
Hey @jciasenza, it looks like there are some commitlint failures left to fix. Please note that we use conventional commits across Open edX projects. You can read about the details [here](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html#type). Can you please amend your commit messages to follow our standard?

**Comment by itsjeyd** [comment]
@jciasenza While this is waiting for another test run, maybe you could bring your changes up-to-date with the `master` branch?

File: src/test/app.test.jsx [hunk 1]
```diff
@@ -9,11 +9,11 @@ import {
   within,
   prettyDOM,
 } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import {
-  initialize,
-  mergeConfig,
-} from '@edx/frontend-platform';
+// import userEvent from '@testing-library/user-event';
```
Comment (deborahgu) [inline src/test/app.test.jsx:12]: Why is this commented out instead of removed?

Comment (jciasenza) [reply src/test/app.test.jsx]: I made a new push to remove those unnecessary lines

**Comment by jciasenza** [comment]
I'm going to close this PR since I realized that it is something that happens only in dev.

**Pull request closed without merging**